src/core/sys/freebsd/sys/event.d

```diff
@@ -28,10 +30,11 @@ enum
     EVFILT_PROC     =  -5, /* attached to struct proc */
     EVFILT_SIGNAL   =  -6, /* attached to struct proc */
     EVFILT_TIMER    =  -7, /* timers */
-    // EVFILT_NETDEV   =  -8, /* no longer supported */
+    EVFILT_PROCDESC =  -8, /* attached to process descriptors */
```

Comment:
I would keep both and add deprecation note to this one (can one deprecate enum field?) so it get's removed in the next release and not break code immediately.